bundles/CoreBundle/Resources/config/pimcore/default.yaml

```diff
@@ -32,10 +32,18 @@ framework:
             main: native://default
             pimcore_newsletter: native://default
     messenger:
+        failure_transport: pimcore_failed_jobs
```

Comment:
yes, this would be my next to define per transport, if we plan to keep in the default.

Reply:
Then we have to split them up from pimcore_core right? Because atm a lot is processed for the core.

Reply:
we could also add docs and keep it optional so you the option to use failed transport by overriding default config like this:
```
pimcore_core: 
  dsn: "doctrine://default?queue_name=pimcore_core"
  failure_transport: pimcore_failed_jobs
```
then it is not a default failed transport so you could still discard failed messages from other & custom transports.

Also then, removing default config from the core and put the example in the docs to define it. wdyt?

Reply:
Sounds good to me! @jdreesen what do you think?

Reply:
Good point @dpfaffenbauer. You are totally right, better be safe than sorry. 
If the queue would get overloaded because of failed jobs, you need to investigate anyways, but it is better to have it in a separate table.

Reply:
Docs added and removed from core default config.